.github/workflows/ci-dgraph-net.yml

```diff
@@ -5,11 +5,6 @@ on:
     branches: 
     - master
   pull_request:
-    types:

```

Comment:
why are we removing these?

Reply:
I though the defaults were fine, but I will undo this to match the dgo action triggers.

Reply:
This helps when a workflow gets stuck.